src/client-side-encryption/auto_encrypter.ts

```diff
@@ -473,8 +474,19 @@ export class AutoEncrypter {
   /**
    * Decrypt a command response
    */
-  async decrypt(response: Uint8Array | Document, options: CommandOptions = {}): Promise<Document> {
-    const buffer = Buffer.isBuffer(response) ? response : serialize(response, options);
+  async decrypt(
```

Comment:
I can make FLE work or we can exempt it like we have to for explain

Reply:
I don't feel strongly.  how much effort do you think it would be to implement for FLE as well?

Reply:
Sorry this wasn't clear, it is working, the changes here make it work but I am offering to defer impacting FLE since originally we _only_ wanted to change the find cursor, I don't mind or feel strongly either way.

Reply:
Yea I think we do, and I think we should get FLE on to the response train 🚂 but I would need to refactor the state machine and the decorate logic a bit more than is comfortable for this PR, so I think we're better off exempting crypto here and we can refactor in a follow up